### ISSUE TITLE
npx ignite-cli without --expo installs expo. Is that expected?

### ISSUE THREAD

**Issue by andrewdang17** [infinitered/ignite#1902]
I'm new to this boilerplate so I ran the vanilla React Native command `npx ignite-cli new PizzaApp` without passing in `--expo`, but I see a bunch of expo packages in package.json and config files referencing expo. Is that suppose to be there?

**Comment by robinheinze** [comment]
@andrewdang17 Yep! Ignite uses [Expo Modules](https://blog.expo.dev/whats-new-in-expo-modules-infrastructure-7a7cdda81ebc) (formerly react-native-unimodules) so that we can utilize the fantastic selection of Expo libraries even in non-expo apps. 

See https://github.com/infinitered/ignite/pull/1830/files
